docs/docs/dev/policies/traffic-log.md

```diff
@@ -152,109 +156,27 @@ conf:
+When `backend ` field is omitted, the logs will be forwarded into the `defaultBackend` of that `Mesh`.
 :::
-::: tab "Universal"
-
-**1. Install Loki**
 
-To install Loki use the instructions on the official [Loki Github repository](https://github.com/grafana/loki/blob/v1.5.0/docs/installation/README.md).
+### Matching
 
-**2. Update the mesh**
-
-The logging backend needs to be configured to send the access logs of your data-planes to `stdout`. 
-Loki will directly retrieve the logs from `stdout` of your containers.
-
-```yaml
-type: Mesh
-name: default
-logging:
-  defaultBackend: loki
-  backends:
-    - name: loki
-      type: file
-      conf:
-        path: /dev/stdout
-```
-    
-**3. Configure Grafana to visualize the logs**
-
-To visualise your **containers' logs** and your **access logs** you need to have a Grafana up and running. 
-You can install Grafana by following the information of the [official page](https://grafana.com/docs/grafana/latest/installation/) or use the one installed with [Traffic metrics](traffic-metrics.md).
-
-<center>
-<img src="/images/docs/loki_grafana_config.png" alt="Loki Grafana configuration" style="width: 600px; padding-top: 20px; padding-bottom: 10px;"/>
-</center>
-
-At this point you can visualize your **containers' logs** and your **access logs** in Grafana by choosing the loki datasource in the [explore section](https://grafana.com/docs/grafana/latest/explore/).
-
-For example, running: `{container="kuma-sidecar"} |= "GET"` will show all GET requests on your cluster.
-To learn more about the search syntax check the [Loki docs](https://grafana.com/docs/loki/latest/logql/).
-:::
-::::
+`TrafficLog` is an [Outbound Connection Policy](how-kuma-chooses-the-right-policy-to-apply.md#outbound-connection-policy).
+For this reason the only supported value for `destinations.match` is `kuma.io/service`.
 
-::: tip
-**Nice to have**
+## Logging external services
 
-Having your Logs and Traces in the same visualisation tool can come really handy. By adding the traceId in your app logs you can visualize your logs and the related Jaeger traces. 
-To learn more about it go read this [article](https://grafana.com/blog/2020/05/22/new-in-grafana-7.0-trace-viewer-and-integrations-with-jaeger-and-zipkin/).
+When running Kuma on Kubernetes you can also log the traffic to external services. To do it, the matched destination section has to have wildcard `*` value.
+In such case `%KUMA_DESTINATION_SERVICE%` will have value `external` and `%UPSTREAM_HOST%` will have an IP of the service.
 
-To set up tracing see the [traffic-trace policy](traffic-trace.md).
-:::
+## Builtin Gateway support
 
-You can also forward the access logs to a collector (such as logstash) that can further transmit them into systems like Splunk, ELK and Datadog.
+Traffic Log is a Kuma outbound connection policy, so Kuma chooses a Traffic Log policy by matching the service tag of the Dataplane’s outbounds.
```

Comment:
```suggestion
`TrafficLog` is a Kuma outbound connection policy, so Kuma chooses a `TrafficLog` policy by matching the service tag of the Dataplane’s outbounds.
```

Reply:
I'm not sure what version of the `data plane` word{s} in this context should be used. Wdyt @michaelbeaumont @parkanzky?